src/main/java/uk/gov/justice/hmpps/prison/service/AppointmentsService.java

```diff
@@ -203,7 +203,7 @@ private String validateLocationAndGetAgency(final String username, final NewAppo
 
         try {
             final var appointmentLocation = locationService.getLocation(newAppointment.getLocationId());
-            final var skipLocationAgencyCheck = authenticationFacade.isOverrideRole("SYSTEM_USER", "GLOBAL_APPOINTMENT");
```

Comment:
The endpoint that calls this code (POST /api/bookings/{bookingId}/appointments) has 2 clients - neither has the SYSTEM_USER role